e2e/playwright/testing-gizmo.spec.ts

```diff
@@ -1,18 +1,18 @@
-import { test, expect } from '@playwright/test'
-
+import { _test, _expect } from './playwright-deprecated'
+import { test } from './fixtures/fixtureSetup'
```

Comment:
Can we revert this name change and do something like

```
import { test, expect } from '@playwright/test'
import { test as fixtureSetupTest } from './fixtures/fixtureSetup'
```

I am confused to why we have a `'./playwright-deprecated'` when we can just do alias importing?